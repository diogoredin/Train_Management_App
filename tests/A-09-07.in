5
1
0
6
3
0
Ã‰vora
Casa Branca
2025-01-01
02:00
1
0
5
1
0
0
