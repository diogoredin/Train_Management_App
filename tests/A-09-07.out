Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: 0|Obi-Wan|FREQUENTE|10|255.00|56:00
1|Yoda|NORMAL|0|0.00|00:00
Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Nome da estação de partida: Nome da estação de destino: Data de partida (YYYY-MM-DD): Hora de partida (HH:MM): 
Itinerário 1 para 2025-01-01 @ 1.36
Serviço #694 @ 1.36
16:57 Évora
17:08 Casa Branca
Número do itinerário: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: 0|Obi-Wan|NORMAL|11|256.15|56:11
1|Yoda|NORMAL|0|0.00|00:00
Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 